microsoft-edge/privacy-whitepaper.md

```diff
@@ -0,0 +1,427 @@
+
+### Block potentially unwanted apps 
+
+Optionally, SmartScreen can check the URL of files you download to see if it’s been categorized as potentially unwanted apps.  Blocking potentially unwanted apps helps deliver more productive, performant, and delightful Windows experiences.  This setting is turned off by default and is only available on Windows 10 devices.  You can turn on **Block potentially unwanted apps** by going to **edge://settings/privacy**.  Learn [how potentially unwanted apps are categorized](https://docs.microsoft.com/windows/security/threat-protection/intelligence/criteria#potentially-unwanted-application-pua) or how to [configure](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-antivirus/detect-block-potentially-unwanted-apps-windows-defender-antivirus) this setting.  
+
+## Spellcheck
+
+Microsoft Edge checks your spelling as you type in the browser.  The spellcheck service is completed locally on the device—Microsoft Edge does not send info about your typing to Microsoft for spellcheck.  You can turn off spellcheck by going to **edge://settings/languages**.  
+
+When you add a new language to Microsoft Edge, the browser will download the new language’s dictionary to the device via HTTPS.  The dictionary is used for the built-in spellcheck service.  You can also add or remove words to your local dictionary by going to **edge://settings/languages**.  Deleting the language from your Microsoft Edge settings will delete the dictionary from the device.  Guest mode will not use the profile’s custom dictionary or any added languages.
+
+## Suggest similar sites
+
+To help resolve URL typos in the address bar that result in a website error, Microsoft Edge may recommend a corrected URL.  When a website navigation error occurs, Microsoft Edge sends the domain of the web address to the Microsoft service to suggest a corrected URL.  Microsoft Edge does not include identifiers or tokens with the domain.  If the service finds a suggestion, it will return the suggested URL.  Microsoft stores the incorrect domain and suggested domain to help improve the service.  To help you navigate to the correct sites, this feature is turned on by default.  To turn off this feature, go to **edge://settings/privacy** and turn off Suggest similar sites when a website can’t be found.
+
+## Sync
+
+Signing in to Microsoft Edge with a Microsoft account allows you to sync your browsing data across all your signed-in versions of Microsoft Edge. You can sync your favorites, settings, form fill data (addresses and more), and passwords.  You must allow permission to turn on sync in Microsoft Edge, and each synced data type can be turned on or off individually.  Favorites includes any swept tabs if you used that feature from the prior version of Microsoft Edge and will sync along with your favorites.   Deleting or modifications to favorites, and other data types from one signed-in version of Microsoft Edge will sync to all
```

Comment:
Be sure that this is an h2 (`## Thank you!`) or higher heading to avoid build issues.

Reply:
made an h2